src/com/tests/helper.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <memory>
+
+#include "cplscheme/CouplingData.hpp"
+#include "cplscheme/SharedPointer.hpp"
+#include "mesh/SharedPointer.hpp"
+
+namespace precice::testing {
+inline cplscheme::PtrCouplingData makeCouplingData(mesh::PtrData data, mesh::PtrMesh mesh, bool exchangeSubsteps)
+{
+  bool requiresInitialization = false;
```

Comment:
```suggestion
inline cplscheme::PtrCouplingData makeCouplingData(mesh::PtrData data, mesh::PtrMesh mesh)
{
  bool requiresInitialization = false;
  bool exchangeSubsteps = true;
```

Reply:
Why not? I think for the serialization tests we will not use `exchangeSubsteps = false` anytime.